pkg/auth/wallet.go

```diff
@@ -27,18 +27,18 @@ func RecoverAddress(message string, signature string) (string, error) {
 }
 
 // For Testing
-func SignMessage(message string) ([]byte, error) {
+func SignMessage(message string) (string, error) {
 	messageByte := []byte(message)
 	messageHash := crypto.Keccak256Hash(messageByte)
 
-	privateKey, err := crypto.HexToECDSA("add your private key")
+	privateKey, err := crypto.HexToECDSA("private key here")
```

Comment:
do you really need to add private key here?

Reply:
This function is not used in API. It's a tooling function for me to generate signature locally for testing. I can read the private from local file instead

Reply:
Okay, please add some comments here.  It looks risky!